qemu/tests/qemu_guest_agent.py

```diff
@@ -793,29 +820,29 @@ class QemuGuestAgentBasicCheckWin(QemuGuestAgentBasicCheck):
     """
     Qemu guest agent test class for windows guest.
     """
-    @error.context_aware
-    def gagent_install(self, params, vm, *args):
+
+    @error_context.context_aware
+    def gagent_install(self, session, params, vm, *args):
```

Comment:
We do not need overwrite this since they have the same behavior after adding `start /wait` to the windows installation command.

Reply:
yes, I updated.